packages/eds-lab-react/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - 2024-03-12
```

Comment:
lets rename this to [unreleased] for now to avoid confusion and then finalize the changelog in the release branch

Reply:
changed to unreleased for now